tilequeue/command.py

```diff
@@ -953,49 +953,71 @@ def tilequeue_prune_tiles_of_interest(cfg, peripherals):
     logger = make_logger(cfg, 'prune_tiles_of_interest')
     logger.info('Pruning tiles of interest')
 
-    logger.info('Fetching tiles of interest ...')
-    tiles_of_interest = peripherals.redis_cache_index.fetch_tiles_of_interest()
-    n_toi = len(tiles_of_interest)
-    logger.info('Fetching tiles of interest ... done. %s found', n_toi)
-
     logger.info('Fetching tiles recently requested ...')
     import psycopg2
 
-    redshift_uri = cfg.yml.get('redshift_uri')
+    prune_cfg = cfg.yml.get('toi-prune', {})
+
+    redshift_uri = prune_cfg.get('redshift-uri')
     assert redshift_uri, ("A redshift connection URI must "
                           "be present in the config yaml")
 
-    redshift_days_to_query = cfg.yml.get('redshift_days')
+    redshift_days_to_query = prune_cfg.get('days')
     assert redshift_days_to_query, ("Number of days to query "
                                     "redshift is not specified")
 
-    tiles_recently_requested = set()
+    new_toi = set()
     with psycopg2.connect(redshift_uri) as conn:
         with conn.cursor() as cur:
             cur.execute("""
                 select x, y, z
                 from tile_traffic_v4
                 where (date >= dateadd(day, -{days}, current_date))
-                  and (z between 0 and 16)
+                  and (z between 10 and 16)
```

Comment:
should `16` here be a configurable **max zoom**? Right now that's 16, but with 2x2 metatiles it'd be 15.